deeplearning4j-nn/src/main/java/org/deeplearning4j/optimize/solvers/accumulation/BasicGradientsAccumulator.java

```diff
@@ -264,4 +264,20 @@ public void reset() {
     public void touch() {
         // no-op
     }
+
+    /**
+     *
+     * @param worker
+     * @return
+     */
+    @Override
+    public int getFreeSpace(int worker) {
+        return Integer.MAX_VALUE;
```

Comment:
Nah, bad design idea, happened before that fancy queue idea discussed with you.  Method will be removed.